### PR TITLE
Media: Fixed image not updating values when resizing

### PR DESCRIPTION
**Pull request by mgbaybay**
## Changes:
- Fixed image not updating values when resizing

Relates to: https://github.com/xibosignage/xibo/issues/3443

### CODE REVIEW

File: lib/XTR/ImageProcessingTask.php [hunk 1]
```diff
@@ -97,7 +97,9 @@ private function runImageProcessing()
             $count++;
 
             // Release image and save
-            $media->release(md5_file($filePath), filesize($filePath));
+            list($updatedImgWidth, $updatedImgHeight) = @getimagesize($filePath);
```
Comment (dasgarner) [inline lib/XTR/ImageProcessingTask.php:100]: Have we explored if there is a more efficient way to do this? For example, does the image processing service know how it has resized the image? Could it return the new width/height for us to use?

**Review by dasgarner — COMMENTED** [review]
I think this would work fine as-is, but I had a question to see if it could be more efficient (maybe not)

File: lib/XTR/ImageProcessingTask.php (hunk 1 above)
Comment (dasgarner) [reply lib/XTR/ImageProcessingTask.php:100]: Sure, but can we avoid calling `getimagesize` twice? If the resizeImage function knows what it resized it to, could that be returned instead?

Comment (mgbaybay) [reply lib/XTR/ImageProcessingTask.php]: Hi again, Dan! I've made a few changes and would like to request a re-review, please. Thanks again :)

File: lib/Entity/Media.php [hunk 2]
```diff
@@ -834,23 +834,21 @@ private function assessDimensions(): void
 
     /**
      * Release an image from image processing
-     * @param $md5
-     * @param $fileSize
+     * @param $img
      */
-    public function release($md5, $fileSize, $height, $width)
+    public function release($img)
```
Comment (dasgarner) [inline lib/Entity/Media.php:839]: Sorry if my comments have confused you. It is better to have explicit parameters as you had it before (you could even add types to them if you wanted).

File: lib/Service/ImageProcessingService.php [hunk 3]
```diff
@@ -62,13 +62,21 @@ public function resizeImage($filePath, $width, $height)
             $img->resize($width, $height, function ($constraint)  {
                 $constraint->aspectRatio();
             });
+
+            // Get the updated height and width
```
Comment (dasgarner) [inline lib/Service/ImageProcessingService.php:66]: Great 👍

**Review by dasgarner — CHANGES_REQUESTED** [review]
The return from resizeImage is exactly what I hoped! That's great.

File: lib/Entity/Media.php (hunk 2 above)
Comment (mgbaybay) [reply lib/Entity/Media.php]: Thanks for this :) I've added a new commit [here ](https://github.com/xibosignage/xibo-cms/pull/2602/commits/542d406e34ee2b22954a661dc8cb7a74531a672e). Requesting review again, please. Thank you!

**Review by dasgarner — APPROVED** [review]

**Comment by ifarzana** [comment]
TEST: /Library END

**Pull request merged**